### ISSUE TITLE
undefined method `scope' for #<ActiveAdmin::Views::Pages::Show:0x007fd6124228c8>

### ISSUE THREAD

**Issue by KelvinMutuma** [activeadmin/activeadmin#4033]
I have added ```scope :failed``` in the **show** block of a resource. 

In the show block, I am showing an index listing or a related model  and I want to use scope to filter out the results using a few toggle buttons on top written ```all```, ```failed``` and ```passed```

How do I go about that?

**Comment by seanlinsley** [comment]
This is what I commonly do:
```ruby
  sidebar 'Associated Records', only: :show do
    attributes_table_for resource do
      row(:users) { strong link_to '[link]', admin_users_path(q: {company_id_eq: resource.id}) }
    end
  end
```

The important part is the `q: {company_id_eq: resource.id}`. This adds a query string on the URL that will be used when visiting the users index page.

**Comment by timoschilling** [comment]
closed as double of #4045
